### PR TITLE
Database performance

### PR DESCRIPTION
**Pull request by marilirulita**
### Database performance audit
**In this branch, we optimize some slow queries in the vet clinic database. The goal is to understand what can impact database performance and find solutions.**
**Here are some screenshots showing before and after the performance:**

count
count_improved

visit-select
visit-select_improved

select_email
select_email_improved

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,165 +1,8 @@
...
-select species.name, count(*) from animals
-JOIN visits ON animals.id=visits.animal
-JOIN vets ON visits.vet=vets.id
-JOIN species ON animals.species_id=species.id
-where vets.name like 'Maisy Smith'
-GROUP BY species.name;
\ No newline at end of file
+ CREATE INDEX animal_count_index on visits(animal_id ASC);
+ CREATE INDEX visits_vet_id_index on visits(vet_id ASC);
+ CREATE INDEX owner_email_index ON owners(email ASC);
```
Comment (enkog) [inline queries.sql]: - [x] Please update the `schema.sql` file with these database schema modifications you made. The `schema.sql` file should only include queries that alter the database's structure, according to the project requirement.

**Review by enkog — CHANGES_REQUESTED** [review]
Hi @marilirulita and @Tadesse-Alemayehu 👋, 

Good job so far! 👍 

✅ Great adding indexes for FOREIGN KEYS in queries as this is one of the basic solutions for the performance problems.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (TadesseDev) [reply queries.sql]: Thanks, @enkog. We will make sure to move those code to the schema file.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi Team, 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**